src/main/filesystem/read-file.ts

```diff
@@ -744,18 +806,26 @@ export function matchLineChromium(line: string): MatchResult | undefined {
   const [pid, timestamp, level, sourceFile] = metadata.split(':');
   const currentDate = new Date();
 
-  // ts format is MMDD/HHmmss.SSS
+  // ts format is MMDD/HHmmss.
+  // https://support.google.com/chrome/a/answer/6271282?hl=en
   // this log format has no year information. Assume that the logs
   // happened in the past year because why would we read stale logs?
-  const [date, time] = timestamp.split('/');
-  const logDate = new Date(
+  const TIMESTAMP_FORMAT_RGX =
+    /^(\d{2})(\d{2})\/(\d{2})(\d{2})(\d{2})\.(\d{3})$/;
+  const parsedTimestamp = TIMESTAMP_FORMAT_RGX.exec(timestamp);
+  if (parsedTimestamp === null) {
+    throw new Error(`Failed to parse Chromium timestamp: ${timestamp}`);
+  }
+  const [, month, day, hour, minute, second, millisecond] = parsedTimestamp;
+  const logDate = new TZDate(
```

Comment:
nit (non-blocking): it looks like we're doing similar logic at 3 points in this file, is it worth DRYing this up into a shared utility method? Or is it just different enough that it's not worth it (looks like this one needs month when another needs year, etc).

Reply:
The webapp and desktop formats are the same, but having a different format for the Chromium one makes the refactoring a bit cumbersome. It's really only 4 added lines each time so I think it's okay to keep it as-is for now.